### PR TITLE
Problem: missing packaging of unreal demo project (fix #86)

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: add release github action

### CODE REVIEW

**Comment by leejw51crypto** [comment]
user can run like this
1. download zip file
2. uncompress
3. open uproject

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          submodules: recursive
+
+      - name: Prepare Plugin
+        run: |
+          cd Plugins/play-unreal-plugin && make 
+
+      - name: Package For Windows,Mac
+        run: |
+          FILENAME="CronosPlayDemo.zip"
+          cd ..
```
Comment (tomtau) [inline .github/workflows/release.yml:31]: will this packaging work with the plugin submodule? should it first init the submodule + run its makefile to download those binaries?

Comment (leejw51crypto) [reply .github/workflows/release.yml:31]: yes, it's submodule initialized and run make in `Prepare Plugin` step

Comment (tomtau) [reply .github/workflows/release.yml:31]: where is `make` being executed in this flow?

**Review by tomtau — APPROVED** [review]

**Pull request merged**